src/main/java/Mentos/components/Ui.java

```diff
@@ -0,0 +1,155 @@
+    }
+
+    /**
+     * Ends the conversation with the user by displaying a farewell message.
+     * This method prints a formatted message to the console, including a
+     * border and a closing statement with a fun reference to Mentos,
+     * encouraging the user to "Pop a Mentos" and stay fresh.
+     */
+    public void endConversation(){
+        print_line();
+        System.out.println("Pop a Mentos, stay fresh! See you next time!");
+        print_line();
+    }
+
+    /**
+     * Prints a formatted message indicating that a Mentos.task event has occurred.
+     *
+     * This method outputs a message to the console that includes the type of event
+     * (such as "todo", "deadline", or "event"), the details of the Mentos.task that was
+     * added, and the number of remaining tasks. The message is enclosed within
+     * a border for visual clarity.
+     *
+     * @param event the type of event that occurred (e.g., "todo", "deadline", "event").
+     * @param task the task object that was added or modified, whose details will be printed.
+     */
+    public void print_remaining_event(String event, Task task, int taskSize){
+        print_line();
+        System.out.printf(event+" Added\n%s\n%d remaining tasks%n",task.toString(),taskSize);
+        print_line();
+    }
+
+    /**
+     * Prints a message indicating that the task has been unmarked as done.
+     *
+     * This method displays a message to the console stating that the given task
+     * has been unmarked and is not completed. It then prints the task's details
+     * and a horizontal line to separate the output.
+     *
+     * @param task The task that has been unmarked as not done.
+     */
+    public void unmarkEvent(Task task){
+        System.out.println("Holdup this Mentos.task is not done!");
+        System.out.println(task.toString());
+        print_line();
+    }
+
+    /**
+     * Prints a message indicating that the task has been marked as done.
+     * This method displays a congratulatory message to the console stating
+     * that the given task has been marked as done. It then prints the task's
+     * details and a horizontal line to separate the output.
+     *
+     * @param task The task that has been marked as done.
+     */
+    public void markEvent(Task task){
+        System.out.println("Nicely done! This Mentos.task is marked as done!");
+        System.out.println(task.toString());
+        print_line();
+    }
+
+    /**
+     * Prints a message indicating that a task has been deleted.
+     *
+     * This method displays a message to the console confirming that the given task
+     * has been deleted. It prints the details of the deleted task and the number
+     * of remaining tasks, followed by a horizontal line to separate the output.
+     *
+     * @param task The task that has been deleted.
+     * @param tasksSize The number of tasks remaining after the deletion.
+     */
+
+    public void deleteEvent(Task task, int tasksSize){
```

Comment:
The name of this method could be slightly misleading as the Javadoc comment mentions that it is deleting a task. But the method is named "deleteEvent". You can consider changing its name to something like "deleteTask"!